Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -257,7 +257,11 @@ - (void)pause:(BOOL)pause controlSentToCall:(OCTCall *)call
             callToUpdate.pausedStatus |= OCTCallPausedStatusByUser;
         }
         else {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wassign-enum"
             callToUpdate.pausedStatus &= ~OCTCallPausedStatusByUser;
+#pragma clang diagnostic pop
+
```

Comment:
That's bad. I've turned off `CLANG_WARN_ASSIGN_ENUM` flag so this won't be an issue anymore. You can rebase on top of `audio` branch and remove this code.